libs/vectorise/include/vectorise/uint/uint.hpp

```diff
@@ -448,7 +449,7 @@ constexpr meta::IfIsUnsignedInteger<T, bool> UInt<S>::operator>=(T other) const
 
 ///////////////////////
 /// unary operators ///
-//////////////////////1/
+///////////////////////
 
 template <uint16_t S>
 constexpr UInt<S> &UInt<S>::operator++()
```

Comment:
My understanding is that `constrexpr`ness of this class (and others alike) is mostly ruined by the `const`ness of `_0`, `_1` and `max`. This morning I've tried to see if this issue can be addressed and found this possible recipe: `_0`, `_1` and `max` should be turned into `constexpr static` member function (and `private` as well, they don't seem to be used anywhere else) that simply return a rvalue.

Reply:
fixing issues with the UInt class is not currently the scope of this PR, and it has taken a bit longer than expected already, however I will gladly do this in a next PR to fix this and similar issues.